backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/bpartner/model/interceptor/C_BP_Group.java

```diff
@@ -51,6 +56,30 @@ public void limitGroupInheritance(@NonNull final I_C_BP_Group bpGroup)
 		{
 			throw new AdempiereException("@Invalid@ @Parent_BP_Group_ID@");
 		}
+	}
+
+	@ModelChange(timings = { ModelValidator.TYPE_BEFORE_NEW, ModelValidator.TYPE_BEFORE_CHANGE }, ifColumnsChanged = I_C_BP_Group.COLUMNNAME_IsDefault)
+	public void unsetPreviousDefault(@NonNull final I_C_BP_Group bpGroup)
```

Comment:
how the following test case will work?

BP1: orgId=0, IsDefault=Y
BP2: orgId=1, IsDefault=N

go to BP2 and set IsDefault=Y. What will happen with BP1?

Reply:
Tested, both groups will keep the isDefault flag, because they are in different orgs.